src/main.py

```diff
@@ -1 +1,6 @@
-print('hello world yashio!')
+# python3.7.5
+
+import os
+
+print(f'hello world yashio!')
```

Comment:
<sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
 F541 f-string is missing placeholders